azure-pipelines.yml

```diff
@@ -203,6 +203,32 @@ stages:
           pathtoPublish: wheelhouse
           artifactName: wheels
 
+  - job: macos_universal2
+    pool:
+      vmImage: 'macOS-12'
+    steps:
+      - task: UsePythonVersion@0
+        inputs:
+          versionSpec: '3.10'
+          architecture: 'x64'
+
+      - bash: |
+          set -o errexit
+          python3 -m pip install --upgrade pip
+          python3 -m pip install cibuildwheel==2.8.0
+        displayName: Install cibuildwheel dependencies
+      - bash: cibuildwheel --output-dir wheelhouse .
+        env:
+          ABI_VERSION: 'cp36'
+          CMAKE_OSX_ARCHITECTURES: 'x86_64;arm64'
```

Comment:
We need this - see https://cmake.org/cmake/help/latest/variable/CMAKE_OSX_ARCHITECTURES.html for details.

Reply:
(Related discussion: https://github.com/pypa/cibuildwheel/pull/704).

Reply:
Why is the `ABI_VERSION` set to `cp36`? I couldn't find a reference for the environment variable in cibuildwheel or cmake.

Reply:
AFAIK it isn't required, it is something I ended up copying from https://github.com/pyca/cryptography/blob/main/.github/workflows/wheel-builder.yml#L93 .

Reply:
Removed :heavy_check_mark: